exporters/committime/app.py

```diff
@@ -46,17 +44,18 @@ def getCollector(
 
 if __name__ == "__main__":
     pelorus.upgrade_legacy_vars()
-    if pelorus.missing_configs(REQUIRED_CONFIG):
-        print("This program will exit.")
-        sys.exit(1)
 
     pelorus.load_kube_config()
     k8s_config = client.Configuration()
     k8s_client = client.api_client.ApiClient(configuration=k8s_config)
     dyn_client = DynamicClient(k8s_client)
 
-    username = os.environ.get("GIT_USER")
-    token = os.environ.get("GIT_TOKEN")
+    username = os.environ.get("GIT_USER", "")
+    token = os.environ.get("GIT_TOKEN", "")
+    if not username and not token:
```

Comment:
How about:
If no GIT_USER or GIT_TOKEN is provided:
   check the git connection, if valid continue with information (meaning it's public repository), otherwise fail saying it could not authenticate and ask to provide valid GIT_USER and GIT_TOKEN?

Reply:
That would have to be done per-repo. I believe authentication errors will already be logged, but I'll have to double check. We could also make a test for that, but that could be done later.

For now, I think logging about it is a good middle ground. Maybe it should be a `warning` instead?

Reply:
@KevinMGranger is currently only one GIT_USER and GIT_TOKEN associated with the exporter for all the repos? If yes then I think it's an issue as USER/TOKEN should be associated with the repo and not all of them. In that case the USER/TOKEN check would be valid as it's 1-1 mapping.

Ideally if the association is one repo - zero or more tokens (even multiple as one git org may have multiple tokens enabled for various periods/users/use-cases).

Reply:
Resolving as per our discussion earlier.